src/domain/event/EventSummaryPage.tsx

```diff
@@ -99,22 +99,30 @@ const EventSummaryPage: React.FC = () => {
   };
 
   const goToEventDetailsPage = () => {
-    eventId && pushWithReturnPath(ROUTES.EVENT_DETAILS.replace(':id', eventId));
+    if (eventId) {
+      pushWithReturnPath(ROUTES.EVENT_DETAILS.replace(':id', eventId));
+    }
   };
 
   const goToCreateOccurrence = () => {
-    eventId && pushWithLocale(ROUTES.CREATE_OCCURRENCE.replace(':id', eventId));
```

Comment:
I don't understand why these were some how bad. I would do it again 😄 That is exactly for what the `&&` -operator is for.

Reply:
Probably the lint warning/error for this was https://eslint.org/docs/latest/rules/no-unused-expressions
So if one returned the result then it wouldn't be an unused expression, now the expression only has a possible side-effect of doing the pushWithLocale if eventId is truthy.